src/Events/Events.jsx

```diff
@@ -0,0 +1,252 @@
+import React, { Component } from 'react';
+import { jsonp } from '../util';
+import i18n from '../i18n';
+import sanitizeHtml from 'sanitize-html';
+import {
+	Card,
+	CardMedia,
+	CardContent,
+	Typography,
+	CardActions,
+	Button,
+	withStyles,
+	Hidden,
+} from '@material-ui/core';
+import Container from '../Container/Container';
+import _ from 'lodash';
+import ResponsiveImage from '../ResponsiveImage/ResponsiveImage';
+import { translate } from 'react-i18next';
+
+// Meetup API test console: https://secure.meetup.com/meetup_api/console/?path=/:urlname/events
+// page=3 = number of results to return in a page, only need the first 3 results
+const GET_UPCOMING_EVENTS_URL =
+	'https://api.meetup.com/Code-Star-Night/events?photo-host=public&page=3&sig_id=226887185&status=upcoming&fields=featured_photo&sig=d035ab8a8f521cbb4ef14eaff79a55f23c3d25eb';
+
+const GET_PAST_EVENTS_URL =
+	'https://api.meetup.com/Code-Star-Night/events?desc=true&photo-host=public&sig_id=226887185&status=past&fields=featured_photo&sig=a60e663f0904424f80fda3b00bf31f315889231c';
+
+const styles = {
+	card: {
+		maxWidth: 300, // 345,
+		margin: '1em',
+		display: 'flex',
+		flexDirection: 'column',
+	},
+	cardBig: {
+		maxWidth: 600, // 345,
+		marginBottom: '3em',
+	},
+	media: {
+		height: 0,
+		paddingTop: '56.25%', // 16:9
+	},
+	content: {
+		flex: '1 0 auto',
+	},
+	section: {
+		position: 'relative',
+		overflow: 'hidden',
+	},
+	nextEventTitle: {
+		color: 'white',
+		fontWeight: 'bold',
+	},
+};
+
+function convertEventResponseToModel(withDescription = false) {
+	return function(mEvent) {
+		const fallbackImage =
+			'https://res.cloudinary.com/codestar/image/upload/v1532409289/codestar.nl/meetup/codestar-night-logo.jpg';
+		const result = {
+			name: mEvent.name,
+			time: mEvent.time,
+			link: mEvent.link,
+			coverUrl: _.get(mEvent, 'featured_photo.photo_link', fallbackImage),
+			withDescription,
+		};
+		if (withDescription) {
+			Object.assign(result, {
+				description: mEvent.description,
+			});
+		}
+		return result;
+	};
+}
+
+@translate(['events'], { wait: true })
+export class Events extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			nextEvents: [],
+			pastEvents: [],
+		};
+		this.renderEventModel = this.renderEventModel.bind(this);
+		this.renderHeader = this.renderHeader.bind(this);
+		this.fetchEvents = this.fetchEvents.bind(this);
+		this.fetchEvents();
+	}
+
+	fetchEvents() {
+		/* Meetup API only allows JSONP for client-side, non authenticated, api key signed GET requests.
+		   must use JSONP conform https://github.com/meetup/api/issues/211
+		   Fetch API does not support JSONP. no-cors mode creates an opaque response without data.
+		*/
+		jsonp(GET_UPCOMING_EVENTS_URL).then(response => {
```

Comment:
Ideally we should fetch things in container components. What do you think?

Reply:
Agreed, will fix in feature/events-page